src/helpers/modules/GraphTranslator.ts

```diff
@@ -64,15 +64,22 @@ function getNameFromIri(iri: string): string {
 }
 
 function addMaps(firstNode: TTGraphData, entity: any, key: string) {
+  let preNode = {
+    name: "middle-node-" + key,
+    iri: "",
+    relToParent: "mapped to",
+    children: [],
+    _children: []
+  }  as TTGraphData;
   entity[key].forEach((nested: any) => {
     Object.keys(nested).forEach(predicate => {
       nested[predicate].forEach((element: any) => {
         if (isObjectHasKeys(element, [Vocabulary.IM.MAPPED_TO])) {
           element[Vocabulary.IM.MAPPED_TO].forEach((mappedTo: any) => {
-            firstNode.children.push({
+            preNode.children.push({
               name: mappedTo.name,
               iri: mappedTo["@id"],
-              relToParent: "mapped to",
+              relToParent: mappedTo.name,
```

Comment:
is that not being used anywhere anymore? is the relationship to parent being stored and displayed?